certbot-dns-google/certbot_dns_google/dns_google.py

```diff
@@ -107,14 +107,22 @@ def add_txt_record(self, domain, record_name, record_content, record_ttl):
 
         zone_id = self._find_managed_zone_id(domain)
 
+        record_contents = self.get_existing_txt_rrset(zone_id, record_name)
+
+        # We need to remove old records and re-add them as a part of the list
+        if record_contents:
+            self.del_txt_record(domain, record_name, record_ttl)
```

Comment:
nit: This update isn't atomic and there's a period of time where all of the TXT records have been deleted. It looks like [the API supports atomic updates](https://cloud.google.com/dns/api/v1/changes#resource). If this is the case, can we modify this request to add deletions to `data` below?

Reply:
Fixed